client/components/chart5y.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
doesn't seem like `react-dom` is used anywhere in this file; should probably not `import` it